packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -106,7 +111,7 @@ export class AtomicBreadcrumbManager implements InitializableComponent {
     return (
       <div class="flex items-center text-sm" part="breadcrumb-wrapper">
         <span class="text-on-background" part="breadcrumb-label">
-          {this.bindings.store.state.facetLabels[facetId] || field}:&nbsp;
+          {this.bindings.store.state.facets[facetId].label || field}:&nbsp;
```

Comment:
What is the difference between `{this.bindings.store.state.facetLabels[facetId]` and `this.bindings.store.state.facets[facetId].label` ?

Reply:
I changed the structure of the store before it contained a map of facet id to facet labels. Now it contains a map of facet id to facet data which right now contains a label and the format.